src/bundler.js

```diff
@@ -72,6 +70,9 @@ var fs = require("fs"),
     cssValidator = require('./css-validator'),
     directoryCrawler = require('./directory-crawler'),
     sourceMap = require('./source-map-utility'),
+    tasks = {
+        minifyJs: require('./tasks/minify/minify-js')

```

Comment:
This file is getting pretty hard to work in - it's about 800 lines. Pulled the minification code out into a separate file. Extracting actions like this into small independent files will make it easier if we ever migrate over to gulp.